docs/DeveloperGuide.adoc

```diff
@@ -295,17 +295,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |mortuary manager |keep track of all bodies and fridges in a single dashboard using the dashboard command |look out for any outstanding work and keep myself up to speed
 
-|`* * *` |user |add a new person |
+|`* * *` |mortuary manager |have a dynamically updated dashboard |reduce errors as compared to updating manually
```

Comment:
Odd phrasing. Could be more specific since UI should be self updating by default - eg. give examples?